implementation/contracts/system/TBTCDepositToken.sol

```diff
@@ -33,13 +35,14 @@ contract TBTCDepositToken is ERC721Metadata, DepositFactoryAuthority {
     }
 
     /// @dev Returns whether the specified token exists.
-    /// @param _tokenId uint256 ID of the token to query the existence of
-    /// @return bool whether the token exists
+    /// @param _tokenId uint256 ID of the token to query the existence of.
+    /// @return bool whether the token exists.
     function exists(uint256 _tokenId) public view returns (bool) {
         return _exists(_tokenId);
     }
 
-    /// @notice           Set allowance for other address and notify.
+    /// @notice           Allow another contract to spend on the caller's behalf.
```

Comment:
Another address, really---no one knows if it's a contract or not (though it may be worth calling out “(generally another smart contract)”, for example).